src/envoy/utils/mixer_control.h

```diff
@@ -31,8 +31,8 @@ void CreateEnvironment(Event::Dispatcher &dispatcher,
                        Runtime::RandomGenerator &random,
                        Grpc::AsyncClientFactory &check_client_factory,
                        Grpc::AsyncClientFactory &report_client_factory,
-                       ::istio::mixerclient::Environment *env,
-                       const Attributes &forward_attributes);
+                       std::string serialized_forward_attributes,
```

Comment:
please use const &

Reply:
TCP control doesn't own the string though? Do you want me to store it there?

Reply:
Yes.

Reply:
Done